src/configuration.js

```diff
@@ -175,6 +207,7 @@ export default class Configuration {
       metrics: options.metrics,
       logger: options.logger,
       tags: options.tags,
+      debugThrottler: throttler,
```

Comment:
What's the rationale for naming the throttler throttler instead of debugThrottler?

Reply:
at the moment we're only using it as a debug throttler but we can always use the throttler to throttle normal sampling decisions too (perhaps if the user is using const sampling and we want to throttle it). So I prefer to leave the name as "throttler" because it'll be a pain in the butt to call this debugThrottler and then rename it in the future.